Source/Core/VideoCommon/PostProcessing.cpp

```diff
@@ -255,15 +255,27 @@ void PostProcessingConfiguration::LoadOptionsConfiguration()
       std::string value;
       ini.GetOrCreateSection(section)->Get(it.second.m_option_name, &value);
       if (!value.empty())
-        TryParseVector(value, &it.second.m_integer_values);
+      {
+        auto integer_values = it.second.m_integer_values;
+        if (TryParseVector(value, &integer_values))
+        {
+          it.second.m_integer_values = integer_values;
+        }
+      }
```

Comment:
I'm not sure I understand why exactly this behaves differently than before. I guess in case of parse failure the previous values are kept instead of deleted, but why would that break the parsing?